app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,125 @@
+      width: 100%; /* Set width to 100% */
+      margin: auto;
+      min-height:200px;
+  }
+
+  /* Hide the carousel text when the screen is less than 600 pixels wide */
+  @media (max-width: 600px) {
+    .carousel-caption {
+      display: none; 
+    }
+  }
+  </style>
+</head>
+<body>
+
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>                        
+      </button>
+      <a class="navbar-brand" href="#" >Logo</a>
+    </div>
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="#">Home</a></li>
+      
+        <li><a href="#">Contact</a></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+<div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <!-- Indicators -
+    <ol class="carousel-indicators">
+      <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+      <li data-target="#myCarousel" data-slide-to="1"></li>
+    </ol>
+
+    <!- Wrapper for slides -->
+    <div class="carousel-inner" role="listbox" >
+      <div class="item active">
+        <img src="https://placehold.it/1200x400?text=IMAGE" alt="Image">
+        <div class="carousel-caption">
+      <img src = "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQIEivg80a5eZiZ9zgpYOQRx5hO-CGdEKEjjRXPcy5WMHvaxGcqhQ"  style=" height :auto; background-size:cover" >
+           
+        </div>     
+      </div>
+
+      <div class="item">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSua0GykLVC7vGBQOLOyw_thHQhAFnev6tNBQTQMAPXY9D3Hv4chQ" alt="Image">
+        <div class="carousel-caption">
+          
+        
+        </div>      
+      </div>
+    </div>
+
+    <!-- Left and right controls -->
+    <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+</div>
+  
+<div class="container text-center">    
+  <h3>Aspects of happiness</h3><br>
+  <div class="row">
+    <div class="col-sm-4">
+      <img src="https://yt3.ggpht.com/-cOP2X6pxo9M/AAAAAAAAAAI/AAAAAAAAAAA/Dn2UTJzgFkU/s900-c-k-no-mo-rj-c0xffffff/photo.jpg" class="img-responsive" style="width:100%" alt="Image">
+      <p style="font-weight:bold;">Meditation</p>
```

Comment:
oye..mjhe ye jo meditation likha h..usme click kru to nya page khule...aisa bnana h...steps bta..ror mai btana